### DISCUSSION TITLE
Users created from the admin are not receiving any email

### DISCUSSION THREAD

**Discussion by MarcusCS** [Sunbird-Lern/Community#7] [Issues] [answered]
Any configuration changes needs to be done in keycloak to send e-mails to new users on creation? Emails are received only for bulk import users.

@rhwarrier @Renuka-S

**Comment by rhwarrier** [comment, marked as answer]
So the bulk creation is working (sending emails) - which scenario is not? 
cc : @reshmi-nair

**Reply by pranavirabatti** [reply]
Yes, for bulk users it is working. We are trying to create user from admin like by calling create user api. In this case we are not receiving any email.  Basically, we are developing a functionality called user management in admin. And we are adding users from this. I think you got the point now. 

@rhwarrier @reshmi-nair @Renuka-S

**Reply by pranavirabatti** [reply]
@rhwarrier @amit-tarento  Any update on this?

**Reply by amitec006** [reply]
if you are using user create API, user will not get any welcome or onboarding notification except in upload API.

**Reply by pranavirabatti** [reply]
@amit-tarento  need to accept a password here to log in? If no password is set for these newly created users then what will be the default password?

**Reply by pranavirabatti** [reply]
In the docs they specified password field is mandatory even then creating user works without a password field.

**Reply by amitec006** [reply]
there is no default password, user can reset his password using forgot password flow.

**Reply by MarcusCS** [reply]
Thanks @amit-tarento @rhwarrier

**Comment by rhwarrier** [comment]
@pranavirabatti  can you also please provide links to the PRD/ design doc (if any) for the functionality being developed? If this is going to be a Sunbird contribution, it'll help to have the Design Council review it. Thanks .
